addons/project/wizard/project_template_create_wizard.xml

```diff
@@ -6,29 +6,36 @@
         <field name="model">project.template.create.wizard</field>
         <field name="arch" type="xml">
             <form string="Project">
-                <div class="oe_title mb-2">
+            <sheet>
+                <div class="oe_title mw-100 mb-2">
                     <label for="name" string="Name"/>
                     <h1>
                         <field name="name" class="o_project_name" placeholder="e.g. Office Party"/>
                     </h1>
                 </div>
-                <group>
                     <group>
                         <field name="date_start" string="Planned Date" widget="daterange" options='{"end_date_field": "date", "always_range": "1"}'/>
                         <label for="alias_name" string="Create tasks by sending an email to" class="pe-2"/>
                         <div class="d-inline-flex">
                             <field name="alias_name" placeholder="e.g. office-party"/>@ <field name="alias_domain_id" placeholder="e.g. mycompany.com" options="{'no_create': True, 'no_open': True}"/>
                         </div>
                     </group>
-                </group>
-                <field name="role_to_users_ids" invisible="not role_to_users_ids">
-                    <list create="0" delete="0" no_open="1" editable="bottom">
-                        <field name="role_id" force_save="1" readonly="1" options="{'no_open': True}"/>
-                        <field name="user_ids" widget="many2many_avatar_user" options="{'no_open': True, 'no_quick_create': True}"/>
-                    </list>
-                </field>
+                    <field name="role_to_users_ids">
+                        <list create="1" delete="1" no_open="1" editable="bottom">
```

Comment:
We don’t need to forcefully pass create and delete. I have explained how it work